tools/proofers/images_index.php

```diff
@@ -214,7 +214,7 @@ function show_delete_all_link( $project, $image_names )
 {
     global $code_url;
 
-    if(($project->state == PROJ_NEW || $project->state == PROJ_P1_UNAVAILABLE) && !empty($image_names))
+    if($project->can_be_managed_by_current_user && ($project->state == PROJ_NEW || $project->state == PROJ_P1_UNAVAILABLE) && !empty($image_names))
```

Comment:
I assume we have a similar check on the actual action, right?

Reply:
I verified by testing that an unprivileged use can see and press the button, but are told they are not authorized to manage the project. I haven't verified that by looking at the code, though.

Reply:
I shoul have been more specific. The above comment refers to the existing code on TEST and PROD. The code in @cpeel's fix-errors sandbox correctly hides the button.